include/dali/c_api.h

```diff
@@ -483,6 +511,43 @@ DLL_PUBLIC const char *daliGetOutputName(daliPipelineHandle *pipe_handle, int id
  */
 DLL_PUBLIC device_type_t daliGetOutputDevice(daliPipelineHandle *pipe_handle, int id);
 
+
+/**
+ * @name Operator traces
+ * @{
+ */
+/**
+ * Checks, if given operator produced a trace with given name.
+ *
+ * @return 0, if the trace with given name exists.
+ */
+DLL_PUBLIC int daliHasOperatorTrace(daliPipelineHandle *pipe_handle, const char *operator_name,
+                                    const char *trace_name);
+
+/**
+ * Returns the traces of the given operator in the DALI Pipeline.
+ *
+ * Operator Traces is a communication mechanism with particular operators in the pipeline. For
+ * more information @see operator_trace_map_t.
+ *
+ * User does not own the returned value. In a situation, when changing of this value is necessary,
+ * user shall copy it to his own memory. The lifetime of this value ends, when the
+ * daliOutputRelease() is called.
+ *
+ * User shall check, if the trace with given name exists (@see daliHasOperatorTrace). If it does
+ * not, the result of this function is undefined. Specifying the name of non-existing operator is
```

Comment:
`undefined`? I'm afraid it would just crash. Undefined means usually just garbage.

Reply:
True. I've added some exception handling, but I've left the `undefined` specification. Although the behaviour right now is pretty defined (returning `nullptr`), I feel that checking the presence of a given trace is a proper usage of the API. That's because we don't want the user to abuse the exception mechanism to check the trace - he should use `daliHasOperatorTrace` function for this.